docs/core/tools/dotnet-dev-certs.md

```diff
@@ -0,0 +1,186 @@
+---
+title: dotnet dev-certs command
+description: The dotnet dev-certs command generates a self-signed certificate to enable HTTPS use in development.
+ms.date: 06/06/2022
+---
+# dotnet dev-certs
+
+**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
+
+## Name
+
+`dotnet dev-certs` - Generates a self-signed certificate to enable HTTPS use in development.
+
+## Synopsis
+
+```dotnetcli
+dotnet dev-certs https 
+  [-c|--check] [--clean] [-ep|--export-path]
+  [--format] [-i|--import] [-np|--no-password]
+  [-p|--password] [-q|--quiet] [-t|--trust]
+  [-v|--verbose] [--version]
+
+dotnet dev-certs https -h|--help
+```
+
+## Description
+
+The `dotnet dev-certs` command generates a self-signed certificate to enable HTTPS use in local web app development. The command recognizes options that perform related functions, such as checking if a certificate is already present, and importing and exporting certificates from and to certificate files.
```

Comment:
The `dotnet dev-certs` command manages a self-signed certificate to enable HTTPS use in local web app development. Its main functions are:
* Generating a certificate
* Trusting a certificate
* Exporting a certificate in various formats so that it can be consumed by other tools.
* Removing the generated certificate from the machine.
* Importing an existing certificate generated by the tool into the machine.